lib/AST/SourceEntityWalker.cpp

```diff
@@ -255,7 +249,7 @@ std::pair<bool, Expr *> SemaAnnotator::walkToExprPre(Expr *E) {
   if (E->isImplicit())
     return { true, E };
 
-  Optional<AccessKind> OpAccess = getAccessKind(E);
+  Optional<AccessKind> OpAccess;
```

Comment:
I'm going to pull the calculation out into a visitor SourceKit can invoke on-demand, instead of doing it in Sema all the time (since SILGen doesn't need it).